lib/Elastica/Exception/ElasticsearchException.php

```diff
@@ -28,15 +30,13 @@ class ElasticsearchException extends \Exception implements ExceptionInterface
     /**
      * Constructs elasticsearch exception.
      *
-     * @param int   $code  Error code
-     * @param array $error Error object from elasticsearch
+     * @param int    $code  Error code
+     * @param string $error Error message from elasticsearch
```

Comment:
This has been string before and is still receiving a string in https://github.com/ruflin/Elastica/pull/1251/files#diff-38e97e4d9ad047da8141046011e72f90R66